libs/ledger-live-common/src/families/vechain/js-synchronisation.ts

```diff
@@ -0,0 +1,91 @@
+import type { GetAccountShape } from "../../bridge/jsHelpers";
+import { BigNumber } from "bignumber.js";
+import { makeSync, makeScanAccounts, mergeOps } from "../../bridge/jsHelpers";
+import { encodeAccountId } from "../../account";
+import eip55 from "eip55";
+import { emptyHistoryCache } from "../../account";
+
+import { getAccount, getLastBlock, getOperations, getTokenOperations } from "./api";
+import { getTokenById } from "@ledgerhq/cryptoassets/tokens";
+
+const getAccountShape: GetAccountShape = async info => {
+  const { initialAccount, currency, derivationMode } = info;
+  let { address } = info;
+  address = eip55.encode(address);
+
+  const oldOperations = initialAccount?.operations || [];
+  const startAt = oldOperations.length ? (oldOperations[0].blockHeight || 0) + 1 : 1;
+
+  const accountId = encodeAccountId({
+    type: "js",
+    version: "2",
+    currencyId: currency.id,
+    xpubOrAddress: address,
+    derivationMode,
+  });
+
+  // get the current account balance state depending your api implementation
+  const { balance, energy } = await getAccount(address);
+
+  // get the current block height
+  const { number: blockHeight } = await getLastBlock();
+
+  // Merge new operations with the previously synced ones
+  const newOperations = await getOperations(accountId, address, startAt);
+
+  //Get last token operations
+  const vthoAccountId = accountId + "+" + encodeURIComponent("vechain/vtho");
+  const VTHOoperations = await getTokenOperations(
```

Comment:
(nitpicking on the case)
```suggestion
  const vthoOperations = await getTokenOperations(
```

Reply:
changed thanks